llms/mlx_lm/generate.py

```diff
@@ -41,17 +41,17 @@ def setup_arg_parser():
         type=str,
         help="Optional path for the trained adapter weights and config.",
     )
-    parser.add_argument(
-        "--trust-remote-code",
-        action="store_true",
-        help="Enable trusting remote code for tokenizer",
-    )
```

Comment:
This argument is annoying to keep remembering to set.. so I just default it to true when using the CLI.